projects/psql.cmake

```diff
@@ -118,10 +118,12 @@ function(build_psql)
       endif()
     endif()
   else()
+    # Put data where it won't be in the STAGE_DIR and copy over what is wanted later
+    set(DATA_DIR ${PSQL_REPO_PATH}/data)
     add_custom_target(psql_build ALL
       COMMENT "Configuring and building psql"
       WORKING_DIRECTORY ${PSQL_REPO_PATH}
-      COMMAND ./configure --prefix=${STAGE_DIR}/psql --libdir=${STAGE_DIR}/lib --includedir ${STAGE_DIR}/include/psql --without-readline
+      COMMAND ./configure --prefix=${STAGE_DIR} --libdir=${STAGE_DIR}/lib --includedir=${STAGE_DIR}/include/psql --datadir=${DATA_DIR} --without-readline
```

Comment:
I tried to clean up the share directory in ovsrpro so I moved some of this stuff out (this gets rid of the locale, et, examples and man folders from the share folder in ovsrpro)